### ISSUE TITLE
Blocks from CC:Tweaked vanishing

### ISSUE THREAD

**Issue by themoiFR** [Laskyyy/Create-Astral#190]
Hello,

I started playing with Astral for a while with two saves : one is survival and one is creative.
Recently, on my survival world, turtles and computers from CC:tweaked disapeared when loading the save. I first thought it was the save that was corrupted because I do not have this issue with my creative save.
So I created a new survival save : same issue. 
I also tryed to reinstall the modpack : nothing changed

I attached the last log, don't know if it can help

[latest.log](https://github.com/Laskyyy/Create-Astral/files/12030653/latest.log)

**Comment by themoiFR** [comment]
Ok so it is going to stay this way ? Not a really big deal, I won't use CC in my maps

**Comment by Giraaffes** [comment]
For anyone who wants a quick fix for this, I've [forked](https://github.com/Giraaffes/cc-restitched) the current version of CC: Restitched and added a check when unloading computers so chunks won't fail to load (like in https://github.com/cc-tweaked/CC-Tweaked/commit/2fe40f669d490a42c84e2377c6e49aaaeeabfc25). A release is available there.

**Comment by MaerkR** [comment]
The mod should probably get removed until this gets fixed, I've spent a lot of time trying to figure out why my contraptions kept disappearing and only came across this now, or at least add a tooltip to CC computers to warn players

**Comment by Erdragh** [comment]
> For anyone who wants a quick fix for this, I've [forked](https://github.com/Giraaffes/cc-restitched) the current version of CC: Restitched and added a check when unloading computers so chunks won't fail to load (like in https://github.com/cc-tweaked/CC-Tweaked/commit/2fe40f669d490a42c84e2377c6e49aaaeeabfc25). A release is available there.

I assume you've put up a Pull Request?

**Comment by Giraaffes** [comment]
> > For anyone who wants a quick fix for this, I've [forked](https://github.com/Giraaffes/cc-restitched) the current version of CC: Restitched and added a check when unloading computers so chunks won't fail to load (like in [cc-tweaked/CC-Tweaked@2fe40f6](https://github.com/cc-tweaked/CC-Tweaked/commit/2fe40f669d490a42c84e2377c6e49aaaeeabfc25)). A release is available there.
> 
> I assume you've put up a Pull Request?

As far as I know you can't have custom mod jars in public CurseForge modpacks. Of course I could make a pull request in CC: Restitched, but I don't think the way I've managed to fix it is a very good way of doing it. Besides, a lot of people have already mentioned this in CC:R issues.

**Comment by Erdragh** [comment]
Being able to put customized jars in the pack depends on the license iirc, so we could try. Also, just opening a PR won't hurt. Worst case scenario it gets rejected. Best case scenario this issue is resolved.

**Comment by Giraaffes** [comment]
> Being able to put customized jars in the pack depends on the license iirc, so we could try. Also, just opening a PR won't hurt. Worst case scenario it gets rejected. Best case scenario this issue is resolved.

That's true. I've opened a pull request now, though if they won't backport the fix from 1.19 I doubt they'll approve it. 

If you can figure out a way to put a custom jar in the pack, that would be brilliant, as computers are basically unusable right now. I have no clue how that would be done though.
